### PR TITLE
LW-6801 miscellaneous mappers for trezor sdk implementation

### PR DESCRIPTION
**Pull request by DominikGuzei**
# Context

Implements misc simple tx sign mappers for the new trezor hw package (https://input-output.atlassian.net/browse/LW-6801)

### CODE REVIEW

File: packages/hardware-trezor/src/transformers/tx.ts [hunk 1]
```diff
@@ -14,14 +14,21 @@ import { mapAuxiliaryData, mapCerts, mapTxIns, mapTxOuts, mapWithdrawals } from
...
-): Promise<Partial<Trezor.CardanoSignTransaction>> => ({
-  auxiliaryData: body.auxiliaryDataHash ? mapAuxiliaryData(body.auxiliaryDataHash) : undefined,
-  certificates: mapCerts(body.certificates ?? [], context),
-  fee: body.fee.toString(),
-  inputs: await mapTxIns(body.inputs, context),
-  outputs: await mapTxOuts(body.outputs, context),
-  withdrawals: mapWithdrawals(body.withdrawals ?? [], context)
-});
+): Promise<Partial<Trezor.CardanoSignTransaction>> => {
+  const certificates = mapCerts(body.certificates ?? [], context);
```
Comment (tomislavhoracek) [inline packages/hardware-trezor/src/transformers/tx.ts:18]: Is there a particular reason you isolated certificates?

File: packages/hardware-trezor/src/TrezorKeyAgent.ts [hunk 2]
```diff
@@ -18,7 +18,7 @@ export class TrezorKeyAgent extends DeprecatedTrezorKeyAgent {
   /**
    * Gets the mode in which we want to sign the transaction.
    */
-  static getSigningMode(tx: Trezor.CardanoSignTransaction): Trezor.CardanoTxSigningMode {
+  static getSigningMode(tx: Omit<Trezor.CardanoSignTransaction, 'signingMode'>): Trezor.CardanoTxSigningMode {
```
Comment (tomislavhoracek) [inline packages/hardware-trezor/src/TrezorKeyAgent.ts:21]: 👍

**Review by tomislavhoracek — APPROVED** [review]
Great work @DominikGuzei 💯 
There is just one minor thing, a question I left

File: packages/hardware-trezor/src/transformers/tx.ts (hunk 1 above)
Comment (DominikGuzei) [reply packages/hardware-trezor/src/transformers/tx.ts:18]: Ah thanks, that's a leftover 👍 will refactor to simpler version

Comment (DominikGuzei) [reply packages/hardware-trezor/src/transformers/tx.ts]: @tomislavhoracek fixed in latest commit

**Pull request merged**